src/snowflake/cli/_plugins/nativeapp/entities/application.py

```diff
@@ -844,7 +844,7 @@ def get_events(
             raise ValueError("first and last cannot be used together")
 
         account_event_table = cls.get_account_event_table()
-        if not account_event_table:
+        if not account_event_table or account_event_table == "NONE":
```

Comment:
We need a test to cover this, as it's part of the contract for event tables.

Reply:
Added them here: https://github.com/snowflakedb/snowflake-cli/pull/1767/files#diff-bf4004353009337dc5b422ed003a0be6ee13a599b6e17236e53afa4b312ec60d